### ISSUE TITLE
enhancement request PF1 attack support

### ISSUE THREAD

**Issue by nerdcore777** [napolitanod/Token-Says#36]
I'd love to see full support for PF1 for this module.
I have a dwarf with a talking axe that needs help  :-)

**Comment by napolitanod** [comment]
Is attack rolling not working for PF1? Last I had tested it was working and I have PF1 specifically addressed in the workflow. For example, here is a simple set up in my test environment for saying something on a claw attack from the Addu in the latest Token Says version.

![image](https://user-images.githubusercontent.com/22696153/145720978-51fcc3a5-ec97-4c6b-a0e5-6a37ed0656dd.png)

**Comment by nerdcore777** [comment]
so I played around with this a fair bit before I made the report, including watching the video a few times. I can get it to work for skills but not for attacks.
here is the skill set up, it works for two tokens (and integrates with polyglot:
![image](https://user-images.githubusercontent.com/95986115/145725861-af65fa64-81a0-4343-a3bb-c4f86466560a.png)
![image](https://user-images.githubusercontent.com/95986115/145725908-dcf01896-5873-4c5d-9d7f-c46713b61eb8.png)

for attacks:
![image](https://user-images.githubusercontent.com/95986115/145725982-8cd24a19-bba2-44bb-a8fe-a7690704ca5f.png)

ingra's attack do not pull from custom rolltable....  does it need to be in a compendium?
